papermill/cli.py

```diff
@@ -171,10 +217,20 @@ def papermill(
         parameters_final.update(read_yaml_file(files))
     for params in parameters_yaml or []:
         parameters_final.update(yaml.load(params, Loader=NoDatesSafeLoader))
-    for name, value in parameters or []:
-        parameters_final[name] = _resolve_type(value)
-    for name, value in parameters_raw or []:
-        parameters_final[name] = value
+    for name, *values in parameters or []:
```

Comment:
This isn't legal syntax in python 2 (which we're still supporting through the end of the year)